examples/bval-evaluation-redeployment/runner/pom.xml

```diff
@@ -47,6 +47,20 @@
     <dependency>
       <groupId>org.apache.cxf</groupId>
       <artifactId>cxf-rt-frontend-jaxrs</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>javax.annotation</groupId>
+          <artifactId>javax.annotation-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>javax.ws.rs</groupId>
+          <artifactId>javax.ws.rs-api</artifactId>
+        </exclusion>
+        <exclusion>
```

Comment:
This seems to be out of the `<exclusions>` section.
I get the following error when I tried to run the example:
cesar:runner [TOMEE-2274---JAXB-bump]$ mvn test
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Malformed POM /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml: Unrecognised tag: 'exclusion' (position: START_TAG seen ...</exclusions>\n        <exclusion>... @142:20)  @ /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml, line 142, column 20
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project org.superbiz:runner:8.0.0-SNAPSHOT (/Users/cesar/git/tomee/examples/bval-evaluation-redeployment/runner/pom.xml) has 1 error
[ERROR]     Malformed POM /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml: Unrecognised tag: 'exclusion' (position: START_TAG seen ...</exclusions>\n        <exclusion>... @142:20)  @ /Users/cesar/git/tomee/examples/bval-evaluation-redeployment/pom.xml, line 142, column 20 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/ModelParseException

Reply:
This was only caused by the parent pom.xml, already resolved